packages/noco-docs/docs/100.data-sources/020.connect-to-data-source.md

```diff
@@ -23,6 +23,14 @@ To connect to an external data source, follow the steps below:
    | Database      | Name of the database to connect to                                                   |
    | Schema name   | Name of the schema to connect to                                                     |
 
+4a. Optionally, if the connection required is TLS/MTLS for MITM protection, follow these additional steps below:
+
+   - Click on `SSL & Advanced Parameters`, \
```

Comment:
1. there is a unncessary `\`
2. `4a.` seems not required

<img width="913" alt="image" src="https://github.com/nocodb/nocodb/assets/35857179/cdc4d760-1575-4c9c-9ace-a2b72713060a">

Reply:
1. ack, removing that extra \
2. 4a. was added considering a reference to image would. I am attaching this image now. 
![image](https://github.com/nocodb/nocodb/assets/113826417/5f1d4427-f4da-48e3-8a55-454680cfab31)